overlays/default.nix

```
@@ -226,12 +229,14 @@ in
 
   openvr_git = callOverride ../pkgs/openvr-git { };
 
-  pkgsx86_64_v2 = makeMicroarch "v2";
-  pkgsx86_64_v3 = makeMicroarch "v3";
-  pkgsx86_64_v4 = makeMicroarch "v4";
+  pkgsx86_64_v2 = makeMicroarchPkgs "x86_64" "x86-64-v2";
+  pkgsx86_64_v3 = makeMicroarchPkgs "x86_64" "x86-64-v3";
+  pkgsx86_64_v4 = makeMicroarchPkgs "x86_64" "x86-64-v4";
 
   pkgsx86_64_v3-core = dropAttrsUpdateScript (import ../shared/core-tier.nix final.pkgsx86_64_v3);
 
+  pkgsznver4 = makeMicroarchPkgs "x86_64" "znver4";
```

Since we're adding this new one, we could scale it up to have them all:

```suggestion
  pkgsAMD64Microarchs =
    (builtins.mapAttrs
      (arch: _inferiors: makeMicroarchPkgs "x86_64" arch)
      (builtins.removeAttrs
        final.lib.systems.architectures.inferiors
        [ "default" "armv5te "armv6" "armv7-a" "armv8-a" "mips32" "loongson2f" ]
      )
    )
```

(then it also needs an update in `maintenance/schemas/packages/inventory.nix` again)